devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -44,6 +44,7 @@ metadata {
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500,0001,FFFF", manufacturer: "ADUROLIGHT", model: "VMS_ADUROLIGHT", deviceJoinName: "ERIA Motion Sensor", mnmn: "SmartThings", vid: "generic-motion-2" //ERIA Motion Sensor V2.0
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500,0001,FFFF", manufacturer: "AduroSmart Eria", model: "VMS_ADUROLIGHT", deviceJoinName: "ERIA Motion Sensor", mnmn: "SmartThings", vid: "generic-motion-2" //ERIA Motion Sensor V2.1
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0001,0003,000F,0020,0500", outClusters: "000A,0019", manufacturer: "frient A/S", model :"MOSZB-140", deviceJoinName: "frient Motion Sensor"
+		fingerprint profileId: "0104", deviceId: "0402", manufacturer: "frient A/S", model :"MOSZB-141", deviceJoinName: "frient Motion Sensor", mnmn: "SmartThingsCommunity", vid: "87753fce-8cd6-3b91-8bde-2483e564252d"
```

Comment:
Please, could You change mnmn to "SmartThings" and add a Raw description in a comment after Your fingerprint ?
... // Raw description: ....

Reply:
Konrad changing the mnmn to `SmartThings` will not work since this is a custom CLI VID.
@frient-design can you add the raw description to the fingerprint

Reply:
> Konrad changing the mnmn to `SmartThings` will not work since this is a custom CLI VID.
> @frient-design can you add the raw description to the fingerprint

ah Yes, then skip that remark.

Reply:
Done